### PR TITLE
when opening new offering form, scroll to its bottom.

### LINKED ISSUE #899: Clicking to Add New Offering Should Navigate User Further Down the Screen
Part of ilios/frontend#4809 in a way.

Steps ...

* Go to somewhere like this ... 
https://ilios3-demo.ucsf.edu/courses/1657/sessions/49971

* image

User should be routed to the bottom of the screen to add the offering details ...

image

Not just to ...

image

### PR DESCRIPTION
**Pull request by stopfstedt**
fixes #899

### CODE REVIEW

File: addon/components/new-offering.js [hunk 1]
```diff
@@ -10,6 +11,10 @@ export default Component.extend({
   courseStartDate: null,
   courseEndDate: null,
   smallGroupMode: true,
+  didRender() {
+    this._super(...arguments);
+    scrollTo('.offering-form .buttons', 1000);
```
Comment (jrjohnson) [inline addon/components/new-offering.js]: `scrollTo` is kind of a hack and will have problems with Octane. Instead use this directly with 
```
import scrollIntoView from 'scroll-into-view';
scrollIntoView(this.element);
```

**Comment by stopfstedt** [comment]
ugh, problems. hanging this off of `didRender()` will cause scrolling down on subsequent re-renderings of this component, e.g. when you toggle from single offering to small groups.
super annoying. 
the better event would be `didInsertElement`, but there's no guarantee that the sub-component we're scrolling down to is in the dom just yet.
i'm taking this back to the bench...

**Comment by jrjohnson** [comment]
Ok. This is a *super* good use of the did-insert octane modifier. So we probably just need to wait a week or two and this might be very easy. I actually think we should consider creating a `scroll-to` modifier if noone else has yet and publish it.

**Comment by stopfstedt** [comment]
ok, cool. if the modifier doesn't pan out, or doesn't land in a timely fashion, then my latest iteration on this seems to check out too.

**Comment by jrjohnson** [comment]
This is unlocked now with our current ember version. Documentation at https://emberjs.github.io/rfcs/0415-render-element-modifiers.html#reusable-helpers

Will need to add the https://github.com/emberjs/ember-render-modifiers as well.

**Comment by stopfstedt** [comment]
superseded by #1245

**Pull request closed without merging**